js/retry-timeout-callback.js

```diff
@@ -250,6 +288,21 @@ var retryTimeoutCallback = (function() {
         return match;
     }
 
+    var __validateEditorRetryOnRetryStep = function(content) {
+        var match = false;
+        try {
+            var pattern = "public class BankService {\\s*" + // readonly boundary
+            "@\\s*Retry\\s*\\(\\s*retryOn\\s*=\\s*TimeoutException\\.class\\s*\\)\\s*" +
```

Comment:
There shouldn't be a `\\s*` before `@` and `Retry`, the annotation only works when it's `@Retry`

Reply:
Done